cypress/integration/mocked/okta_change_password.spec.ts

```diff
@@ -0,0 +1,322 @@
+/**
+ * Test Scenarios:
+ *
+ * Description                                      | Outcome
+ * -------------------------------------------------|-------------------------------------------------------------------
+ * State token has expired                          | show session expired page
+ * State token invalid, recovery token invalid      | show link expired page
+ * State token invalid, recovery token valid        | show reset password page with global error: There was a problem changing your password, please try again.
+ * State token absent, recovery token invalid       | show link expired page
+ * State token absent, recovery token valid         | show reset password page with global error: There was a problem changing your password, please try again.
+ * Unexpected error checking the token              | show link expired page
+ * Unexpected error changing the password           | show password page with global error: There was a problem changing your password, please try again.
+ * Password is too short                            | show password page with field error: Please make sure your password is at least 8 characters long.
+ * Password is too long                             | show password page with field error: Please make sure your password is not longer than 72 characters.
+ * Password cannot be your current password         | show reset password page with field error: Please use a password that is different to your current password.
+ * Password cannot be a common or breached password | show reset password page with field error: Please use a password that is hard to guess.
+ * Password set successfully                        | show password updated confirmation page
+ */
+
+describe('Change password in Okta', () => {
+  context.skip('reset password page', () => {
```

Comment:
just checking, is the skip there until we turn on Okta or was it left over from testing?

Reply:
By skipping these tests if there's a breaking change in the intervening time until we enable Okta could we potentially miss it?